libs/shared/contentful/src/lib/contentful.client.config.ts

```diff
@@ -19,4 +19,7 @@ export class ContentfulClientConfig {
 
   @IsString()
   public readonly graphqlEnvironment!: string;
+
+  @IsString()
+  public readonly cacheTTL?: string;
```

Comment:
Could we update the [fxa-auth-server](https://github.com/mozilla/fxa/blob/ba515e81a3abb616980e2f58fe4504a5b594eb6c/packages/fxa-auth-server/config/index.ts#L1989) and [fxa-graphql-api](https://github.com/mozilla/fxa/blob/ba515e81a3abb616980e2f58fe4504a5b594eb6c/packages/fxa-graphql-api/src/config.ts#L44) config's with this new env var as well?

Reply:
Also would it make sense to change this to `number` instead of string? Save us the parseInt

Reply:
We can, but it's not necessary. It's optional and defaults to 5 minutes as per the ticket reqs, so figured no need to add it as an env var.

Reply:
> Also would it make sense to change this to number instead of string? Save us the parseInt

Probably so, I made it a string originally since all env vars are strings, so we have to parseInt at some point. Probably best to parseInt when we're ingesting the env vars - will change.